src/moj/components/multi-file-upload/multi-file-upload.mjs

```diff
@@ -12,81 +10,89 @@ export function MultiFileUpload(params) {
   }
 
   this.defaultParams = {
-    uploadFileEntryHook: $.noop,
-    uploadFileExitHook: $.noop,
-    uploadFileErrorHook: $.noop,
-    fileDeleteHook: $.noop,
+    uploadFileEntryHook: () => {},
+    uploadFileExitHook: () => {},
+    uploadFileErrorHook: () => {},
+    fileDeleteHook: () => {},
     uploadStatusText: 'Uploading files, please wait',
     dropzoneHintText: 'Drag and drop files here or',
     dropzoneButtonText: 'Choose files'
   }
 
-  this.params = $.extend({}, this.defaultParams, params)
-  this.container = $(this.params.container)
+  this.params = Object.assign({}, this.defaultParams, params)
 
-  this.container.addClass('moj-multi-file-upload--enhanced')
+  this.container = this.params.container
+  this.container.classList.add('moj-multi-file-upload--enhanced')
 
-  this.feedbackContainer = this.container.find(
+  this.feedbackContainer = this.container.querySelector(
     '.moj-multi-file__uploaded-files'
   )
+
   this.setupFileInput()
   this.setupDropzone()
   this.setupLabel()
   this.setupStatusBox()
-  this.container.on(
-    'click',
-    '.moj-multi-file-upload__delete',
-    this.onFileDeleteClick.bind(this)
-  )
+
+  this.container.addEventListener('click', this.onFileDeleteClick.bind(this))
 }
 
 MultiFileUpload.prototype.setupDropzone = function () {
-  this.fileInput.wrap('<div class="moj-multi-file-upload__dropzone" />')
-  this.dropzone = this.container.find('.moj-multi-file-upload__dropzone')
-  this.dropzone.on('dragover', this.onDragOver.bind(this))
-  this.dropzone.on('dragleave', this.onDragLeave.bind(this))
-  this.dropzone.on('drop', this.onDrop.bind(this))
+  this.dropzone = document.createElement('div')
+  this.dropzone.classList.add('moj-multi-file-upload__dropzone')
+
+  this.dropzone.addEventListener('dragover', this.onDragOver.bind(this))
+  this.dropzone.addEventListener('dragleave', this.onDragLeave.bind(this))
+  this.dropzone.addEventListener('drop', this.onDrop.bind(this))
+
+  this.fileInput.replaceWith(this.dropzone)
+  this.dropzone.appendChild(this.fileInput)
```

Comment:
Does this work in this order? I assume it you've checked and it does, but on first reading it doesn't seem like it should! If fileInput has been replacedWith can it be appended?